exporters/trace/src/main/java/com/google/cloud/opentelemetry/trace/TraceTranslator.java

```diff
@@ -210,8 +209,8 @@ static Links toLinksProto(
   private static Link toLinkProto(io.opentelemetry.sdk.trace.data.SpanData.Link link) {
     checkNotNull(link);
     return Link.newBuilder()
-        .setTraceId(link.getContext().getTraceId().toLowerBase16())
-        .setSpanId(link.getContext().getSpanId().toLowerBase16())
+        .setTraceId(link.getContext().getTraceIdAsHexString())
```

Comment:
We may need to confirm that the new trace/span ID format will be accepted by Cloud Trace.
Could you manually test that via an example against Cloud Trace?

Reply:
Yep ran `TraceExporterExample.java` with the current exporter-trace dependency and it was sent successfully to Cloud Trace:
![image](https://user-images.githubusercontent.com/8233863/96096564-b4653f80-0f1b-11eb-872f-33a2835d21a6.png)